script/sheet/actor.js

```diff
@@ -78,14 +78,35 @@ export class DarkHeresySheet extends ActorSheet {
     await prepareCommonRoll(rollData);
   }
 
+  _getCharacteristicOptions (selected) {
+    const characteristics = []
+    for (let char of Object.values(this.actor.data.data.characteristics)) {
+      characteristics.push({
+        label: char.label,
+        base: char.base,
```

Comment:
Ah, I think this is the bug. This should be `base: char.total` (though I'd recommend renaming `base` to be `target`).

`char.base` is the starting point for characteristics, determined during character generation. It's then increased during level up by an amount, `char.advance`, so base+advance = total.

Reply:
Oh good to know! I'll get that fixed today, thanks for finding it.

Reply:
Resolved 👍